src/components/with-organization/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ComponentType, memo } from 'react';
```

Comment:
Vi har allerede with-organizations i redux, vi trenger ikke å ha en egen store for en enkelt organization. Da er vi tilbake med å håndtere at store må slettes mellom hver side-navigering. Kan bare opprette en selector-funksjon for en enkelt organisasjon i OrganizationsReducer-store som returnerer akkurat den du vil ha.

Reply:
Det ser ut som "with-organizations" komponent har ansvar for kommunikasjon med organizations-bff og ikke organization-catalogue

Reply:
Ja, det er sant, de er forskjellige. Kan likevel vurdere å lagre hele organization-catalogue i redux-store og ha en selector-funksjon for å hente enkeltvise organisasjoner.